lib/checks/label/label-content-name-mismatch.js

```diff
@@ -34,15 +34,17 @@ function isStringContained(compare, compareWith) {
 }
 
 /**
- * Curate given text, by removing emoji's, punctuations, unicode and trimming.
+ * Curate given text, by removing emoji's, punctuations, unicode and trim whitespace.
  *
  * @param {String} str given text to curate
  * @returns {String}
  */
 function curateString(str) {
-	return text.sanitize(
-		text.replacePunctuation(
-			text.replaceNonBmpUnicode(text.replaceEmojiUnicode(str))
-		)
+	const noEmojiUnicodeStr = text.replaceUnicode(str, (type = 'emoji'));
```

Comment:
Why not:

```js
text.replaceUnicode(str, {
  emoji: true,
  nonBmp: true,
  punctuations: true
})
```

Reply:
Firstly, unicode only comprises `emoji` and `nonBmp` at the moment. Punctuations are handled separately. I wouldn't want to mix and match everything into the same options object as it can get verbose. 

Hence went with the simpler `type=*` argument.

Reply:
After discussion, updated as agreed.